taxi/views.py

```diff
@@ -104,16 +160,19 @@ class DriverLicenseUpdateView(LoginRequiredMixin, generic.UpdateView):
 
 class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
     model = Driver
-    success_url = reverse_lazy("")
+    success_url = reverse_lazy("taxi:driver-list")
 
 
 @login_required
 def toggle_assign_to_car(request, pk):
+    current_url = request.META["HTTP_REFERER"]
     driver = Driver.objects.get(id=request.user.id)
+
     if (
         Car.objects.get(id=pk) in driver.cars.all()
     ):  # probably could check if car exists
```

Comment:
It would be beneficial to check if the `Car` object exists before attempting to access it. This can prevent potential errors if an invalid `pk` is provided.